src/core/lv_obj.h

```diff
@@ -399,6 +399,7 @@ void lv_obj_set_id(lv_obj_t * obj, void * id);
 void * lv_obj_get_id(const lv_obj_t * obj);
 
 /**
+ * DEPRECATED IDs are used only the print the widget trees. To find a widget use obj_name
```

Comment:
```suggestion
 * DEPRECATED IDs are used only to print the widget trees. To find a widget use obj_name
```
Also any reason to not put the full name of the function here ? `lv_obj_find_by_name`

Reply:
Thanks, fixed!